tests/time_tests/README.md

```diff
@@ -0,0 +1,25 @@
+# Time Tests
+
+This test suite contains pipelines, which are executables. The
+pipelines measure the time of their execution, both total and 
+partial. A Python runner calls the pipelines and calcuates the 
+average execution time.
```

Comment:
@asomsiko I propose to change `average` to `aggregated` to give us more freedom in metrics changes

Reply:
The Readme is for current state. The runner does display average. It also does many other things which we may explain another PR.